website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,176 @@
+
+### Upgrading nodes
+
+A straight-forward way to upgrade nodes that are managed by Karpenter is to add new upgraded nodes, move pod workloads from the old nodes to the new ones, and delete the old nodes.
+Another way to do upgrades is to set `ttlSecondsUntilExpired` values so old nodes are deleted after a set period of time, so they can then be replaced with newer nodes.
+For details on upgrading nodes with Karpenter, see [Upgrading nodes with Karpenter](/docs/tasks/upgrade-nodes.md) for details.
+
+### Cluster operator concepts
+
+Understanding the following concepts will help you in carrying out the tasks just described.
+
+#### Constraints
+
+The concept of layered constraints is key to using Karpenter.
+With no constraints defined in provisioners and none requested from pods being deployed, Karpenter on AWS is free to choose from the entire universe of features available from EC2.
+Nodes can be created using any instance type and run in any zones.
+On-demand or spot instances can be used.
+
+Most of the same constraints a Karpenter cluster operator can set up on a provisioner can be used by an application developer in a podspec to request particular features from a node.
+Refer to the description of Karpenter constraints in the Application Developer section below for details.
+
+#### Scheduling
+
+To avoid conflicts, Karpenter only schedules pods that the Kubernetes scheduler has marked unschedulable.
+Karpenter optimistically creates the Node object and binds the pod.
+This stateless approach helps to avoid race conditions and improves performance.
+If something is wrong with the launched node, Kubernetes will automatically migrate the pods to a new node.
+
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+This is useful if there is additional room in the node due to imperfect packing shape or because workloads finish over time.
+
+#### Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to eventually work with other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set some values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+
+Choosing spot instances is one feature that is currently specific to AWS, but one that could be generalized to other providers.
+Furhter, cloud providers may choose to implement support for additional vendor specific labels like `node.k8s.aws/capacity-type`.
+See [AWS labels](/docs/cloud-providers/aws/) for details.
```

Comment:
The project has evolved since we spoke :). This is now generically available as `karpenter.sh/capacity-type`
```suggestion
```